### PR TITLE
Activate google analytics and gtag

### PR DESCRIPTION
**Pull request by da-kami**
See: https://v2.docusaurus.io/docs/using-plugins/#docusaurusplugin-google-analytics

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -6,7 +6,14 @@ module.exports = {
   favicon: 'img/favicon.ico',
   organizationName: 'coblox', // Usually your GitHub org/user name.
   projectName: 'comit-network', // Usually your repo name.
+  plugins: ['@docusaurus/plugin-google-analytics', '@docusaurus/plugin-google-gtag'],
```
Comment (bonomat) [inline docusaurus.config.js]: I might be confused but I think we only need one :)
Imho we only want `@docusaurus/plugin-google-gtag`. The other one was the _old way_ of tracking. I'm not 100% sure, so maybe just use one for now and give it a try?

Comment (bonomat) [inline docusaurus.config.js]:
```suggestion
  plugins: ['@docusaurus/plugin-google-gtag'],
```

File: docusaurus.config.js [hunk 2]
```diff
@@ -6,7 +6,14 @@ module.exports = {
...
   themeConfig: {
+    googleAnalytics: {
+      trackingID: 'G-RQEDN1PVTD',
+    },
+    gtag: {
+      trackingID: 'G-RQEDN1PVTD',
+    },
```
Comment (bonomat) [inline docusaurus.config.js]:
```suggestion
    gtag: {
      trackingID: 'G-RQEDN1PVTD',
    },
```

**Comment by D4nte** [comment]
To me @bonomat's input should be enough to move forward on this topic :)

**Comment by da-kami** [comment]
> I might be confused but I think we only need one :)
> Imho we only want @docusaurus/plugin-google-gtag. The other one was the old way of tracking. I'm not 100% sure, so maybe just use one for now and give it a try?

You are right, thanks for the hint. We can configure general tags (from templates) to achieve the same functionality as with GA. Source:
https://www.analyticsmania.com/post/google-tag-manager-vs-google-analytics/

**Comment by da-kami** [comment]
Note that I updated the tag after getting started with GTag (activated account) because it gave me a different container ID. (still a lot to learn...)

**Pull request merged**